api/src/api/v1/prerequisites/require_child_user.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * "user_details-child" route pre-requisite
+ *
+ * Determines if the authenticated user is trying to access an user
+ * which is actually a "child" entity.
+ *
+ * Conditions under which this is true for user X attemptying to access:
+ * user Y
+ * - User X is an admin for a community business for which user Y is a VISITOR
```

Comment:
Does this return true for siblings too?
ie 

- User X is an volunteer admin for a community business for which user Y is a volunteer admin

Reply:
no, only CB admins, check the is_child_user pre-req

Reply:
Can this be specified in the description